charts/nebraska/templates/secrets.yaml

```
@@ -12,6 +13,12 @@ data:
   {{- if $useDbPassword }}
   dbPassword: {{ (tpl .Values.config.database.password .) | b64enc }}
   {{- end }}
+  {{- if $useOidcAuth }}
+  oidcClientID: {{required "A valid 'clientId' is required when using oidc authentication" .Values.config.auth.oidc.clientID |toString | b64enc }}
+  oidcClientSecret: {{required "A valid 'clientSecret' is required when using oidc authentication" .Values.config.auth.oidc.clientSecret |toString | b64enc }}
+  oidcSessionAuthKey: {{required "A valid 'sessionAuthKey' is required when using oidc authentication" .Values.config.auth.oidc.sessionAuthKey |toString | b64enc }}
+  oidcSessionCryptKey: {{required "A valid 'sessionCryptKey' is required when using oidc authentication" .Values.config.auth.oidc.sessionCryptKey |toString | b64enc }}
```

```suggestion
  oidcClientID: {{ required "A valid 'clientId' is required when using oidc authentication" .Values.config.auth.oidc.clientID | toString | b64enc }}
  oidcClientSecret: {{ required "A valid 'clientSecret' is required when using oidc authentication" .Values.config.auth.oidc.clientSecret | toString | b64enc }}
  oidcSessionAuthKey: {{ required "A valid 'sessionAuthKey' is required when using oidc authentication" .Values.config.auth.oidc.sessionAuthKey | toString | b64enc }}
  oidcSessionCryptKey: {{ required "A valid 'sessionCryptKey' is required when using oidc authentication" .Values.config.auth.oidc.sessionCryptKey | toString | b64enc }}
```
Add spaces to be consistent across existing code style